style.css

```diff
@@ -23,43 +23,32 @@ header {
   background: no-repeat center/100% url(images/headerbg.svg);
 }
 
-.nav-el {
-  display: flex;
-  justify-content: space-between;
-}
-
 a {
   text-decoration: none;
 }
 
-.my-logo {
-  font-size: 20px;
-  font-weight: 700;
-  line-height: 20px;
-  letter-spacing: 0;
-  margin-top: 1rem;
+.navbar {
+  min-height: 80px;
   display: flex;
-  justify-content: left;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 25px;
 }
 
-.my-logo a {
-  color: #6070ff;
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 60px;
 }
 
-.navigation {
-  list-style-type: none;
-  display: flex;
-  gap: 2rem;
-  margin-left: auto;
-  margin-right: 1rem;
+.nav-branding {
+  font-size: 2rem;
 }
 
 .nav-link {
-  display: flex;
-  text-decoration: none;
+  transition: 0.5s ease;
```

Comment:
- [ ] Kindly make sure to follow the given Figma design. Your mobile menu doesn't look like the given in requirements.
- [ ] The menu should follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, tests, margins).

https://user-images.githubusercontent.com/70269507/173587251-9996cc9c-cd6a-44ee-a035-41073bb89761.mov

- [ ] For menu background you have to use backdrop and blur. 
- [ ] Kindly also remove the list bullet points from the menu with display: none;